backend/api/v1/views.py

```diff
@@ -47,13 +47,96 @@ def show_user(user):
     return jsonify(return_dict)
 
 
+@blueprint.route("/course", methods=["GET"])
+def show_courses():
+    """
+    Get information on the all courses
+    """
+    return_dict = [
+        {
+            "courseSlug": course.slug,
+            "course": course.name,
+            "contentGroups": [
+                {
+                    "contentGroupSlug": content_group.slug,
+                    "contentGroup": content_group.name,
+                    "contents": [
+                        {
+                            "slug": content.slug,
+                            "title": content.title,
+                            "description": content.description,
+                        }
+                        for content in content_group.contents
+                    ],
+                }
+                for content_group in course.content_groups
```

Comment:
I guess that's fine for the current state of the project. On the long term we should see if this could be covered by a utility function that can be called instead of creating the (close to same) objects in all the endpoints. But that can also be done later.